src/components/AppHeader.tsx

```diff
@@ -1,14 +1,29 @@
+import { useIsMobile } from "../assets/customHooks/useIsMobile"
 
 const AppHeader = () => {
+    const isMobile = useIsMobile()
+
     return (
-        <div className="flex justify-between">
-            <div>hamburger</div>
-            <div>logo</div>
-            <nav className="flex">
-                <div>S</div>
-                <div>P</div>
-                <div>B</div>
-            </nav>
+        <div className="header-container flex align-center">
+            {isMobile && <img src="src/assets/img/HAMBUR.svg" alt="hamburger-menu" />}
+            <div className="flex">
+                <img className="logo" src="src/assets/img/about-logo@3x 1.svg" alt="logo" />
+                {!isMobile &&
+                    <div className="left-nav-bar flex align-center">
+                        <h1 className="main-title">EPICURE</h1>
+                        <h2 className="secondary-title">Restaurants</h2>
+                        <h2 className="secondary-title">Chefs </h2>
```

Comment:
instead of rendering 3 images, think about putting it inside an array and loop through it
-- try to always think about what would happen if tomorrow / 2 weeks from now u'll have to add more buttons/icons/navigation, this will require you to add more tags instead of just adding another element into an array nad loop through.